### DISCUSSION TITLE
Tablet UI: Try what GNOME couldn't?

### DISCUSSION THREAD

**Discussion by imxade** [pop-os/cosmic-epoch#869] [Ideas]
Would be great to have a Touch Screen UI, sufficient to carry out mostly stuff without relying on keyboard.

**Comment by usbeehu** [comment]
That would be nice, but it shouldn't be a priority for now. Maybe on Cosmic 2 (or whatever version scheme they will use).

**Comment by sachinchaudhary1808** [comment]
that would be very nice cosmic everywhere
